libnavui-app/src/main/java/com/mapbox/navigation/ui/app/internal/controller/RouteStateController.kt

```diff
@@ -19,6 +27,51 @@ class RouteStateController(store: Store) : StateController() {
     override fun onAttached(mapboxNavigation: MapboxNavigation) {
         super.onAttached(mapboxNavigation)
         this.mapboxNavigation = mapboxNavigation
+
+        coroutineScope.launch {
```

Comment:
Is it possible to extract this logic to a separate controller or component? I believe it will give us an option to unregister it, when user decides to disable alternative routes in the settings. If you think there should be a better option to handle this on Drop-In side, then we can merge it this way and keep the option always enabled for now.

Reply:
Moved it to a separate `RouteAlternativeStateController`.